python/rfnoc_modtool/rfnoc-newmod/cmake/Modules/Findfpga.cmake

```diff
@@ -3,13 +3,13 @@ PKG_CHECK_MODULES(PC_FPGA fpga)
 
 FIND_PATH(
     UHD_FPGA_DIR
-    NAMES /usrp3/top/Makefile.common
+    NAMES /usrp3_rfnoc/top/Makefile.common
     HINTS $ENV{PYBOMBS_PREFIX}/src/uhd-fpga
         ${PC_FPGA_INCLUDEDIR}
-    PATHS ${CMAKE_INSTALL_PREFIX}/src/uhd-fpga/usrp3
+        ~/src/uhd/fpga-src
```

Comment:
I think I thought of adding this once but this assumes that the user is using a prefix called 'src', when it can actually be whatever he/she wants. So this would help in specific cases only.